addOns/ascanrulesAlpha/src/main/javahelp/org/zaproxy/zap/extension/ascanrulesAlpha/resources/help/contents/ascanalpha.html

```diff
@@ -75,6 +75,15 @@ <H2>Spring4Shell (CVE-2022-22965)</H2>
 <p>
 Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/main/addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/Spring4ShellScanRule.java">Spring4ShellScanRule.java</a>
 
+<H2>Server Side Template Injection</H2>
+This rule attempts to detect situations in which user input might be interpreted as part of the template and processed on the server, versus the user input simply being used as an argument to the template/engine.
+
```

Comment:
Add `<p>` here as well for consistency with latest changes? (Same for following entry.)

Reply:
On it